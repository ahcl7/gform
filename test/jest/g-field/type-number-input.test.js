import { generateComponent } from "./common";

describe('GField with type number input', function () {
  it('should render correctly', () => {
    const component = generateComponent({ key: 'nr', type: 'input@number', flex: 'md-6', addable: true });
    expect(component.html()).toMatchSnapshot();
  });
});

