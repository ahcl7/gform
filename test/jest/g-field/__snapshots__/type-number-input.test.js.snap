// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type number input should render correctly 1`] = `
<div class="col-flex md-6 px-2">
  <div class="g-tf-wrapper g-tf  r">
    <div class="input">
      <input autocomplete="off"
             class="g-tf-input"
             type="number"
             label="Nr"
      >
    </div>
    <div class="g-tf-append__inner">
    </div>
  </div>
</div>
`;
